src/asmcnc/skavaUI/screen_yeticut_lobby.py

```diff
@@ -22,8 +24,17 @@
                 padding: scaling_utils.get_scaled_tuple((dp(10), dp(10)))
 
                 Button:
-                    text: 'Back up ^'
                     on_press: root.go_back()
+                    background_color: color_provider.Colors['transparent']
+                    FloatLayout:
+                        size: self.parent.size
+                        pos: self.parent.pos
+                        Image:
+                            source: paths.join(paths.SKAVA_UI_IMG_PATH, "shapes_lobby_back_button.png")
```

Comment:
Sorry, I should've been clearer. You can use paths.get_resource(file_name) and it will be returned 🙂

Reply:
You might even be able to use the native kivy resource_find function in the Builder but not certain.

Reply:
Probs best to use paths.get_resource for consistency anyway